README.plugin.md

```diff
@@ -7,6 +7,18 @@
 See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin)
 for how to install Foreman plugins
 
+## Migrating your plugin:
```

Comment:
> it shouldn't be prominent in users docs.

That was my original thought. So what would you suggest? My last resort would be [wiki](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin).